core/examples/luigi-sample-angular/e2e/tests/luigi-client-features.spec.js

```diff
@@ -107,6 +107,29 @@ describe('Luigi client features', () => {
             expect(loc.hash).to.eq('#/projects/pr2');
           });
         });
+
+      // check if path exists
+      cy.go(-2);
```

Comment:
I tend to not like relying on browser history, but since the flows in here rely on eachother it's ok.